apps/desktop-wallet/src/features/send/CheckAddressesBox.tsx

```diff
@@ -29,17 +30,20 @@ const CheckAddressesBox = ({ fromAddress, toAddressHash, dAppUrl, ...props }: Ch
       </AddressRow>
 
       {(toAddressHash || dAppUrl) && (
-        <AddressRow>
-          <AddressLabel>{t('To')}</AddressLabel>
+        <>
+          <HorizontalDivider secondary />
+          <AddressRow>
+            <AddressLabel>{t('To')}</AddressLabel>
 
-          {toAddressHash && (
-            <DestinationAddress>
-              <AddressBadge addressHash={toAddressHash} truncate appendHash withBorders />
-              <ExplorerLink addressHash={toAddressHash} />
-            </DestinationAddress>
-          )}
-          {dAppUrl && <DestinationAddress>{dAppUrl}</DestinationAddress>}
-        </AddressRow>
+            {toAddressHash && (
+              <DestinationAddress>
+                <AddressBadge addressHash={toAddressHash} truncate appendHash withBorders />
+                <ExplorerLink addressHash={toAddressHash} />
+              </DestinationAddress>
+            )}
+            {dAppUrl && <DestinationAddress>{dAppUrl}</DestinationAddress>}
+          </AddressRow>
```

Comment:
I see that you made these changes here, but they are not reflected properly in the rest of our send modals. I'll take care of it

<img width="350" alt="image" src="https://github.com/user-attachments/assets/6a28860a-7372-4ffd-8f3e-0bd06459b5fd" />

<img width="350" alt="image" src="https://github.com/user-attachments/assets/e1c2ee01-c466-4484-baaf-06e84be630bb" />